doc/user-guide/cli.md

```diff
@@ -64,4 +64,29 @@ Then, execute:
 
 ```sh
 ploomber-cloud deploy
-```
\ No newline at end of file
+```
+
+## Defining environment variables
```

Comment:
don't forget to mention that the `.env` file is not uploaded and that's replaced by an empty file at runtime for security reasons

Reply:
[Added a mention for this](https://github.com/ploomber/doc/pull/98/files#diff-bdba8f4b461f494edc3b95709bb0d5d869d0d6d1593c34c1ac790c9f24787e64R87-R88).